extensions/amp-form/0.1/amp-form.js

```diff
@@ -189,10 +267,17 @@ function installSubmissionHandlers(win) {
 }
 
 
+/**
+ * @param {!Window} win
+ * @private
+ */
 function installAmpForm(win) {
   return getService(win, 'amp-form', () => {
     if (isExperimentOn(win, TAG)) {
-      installStyles(win.document, CSS, () => installSubmissionHandlers(win));
+      installStyles(win.document, CSS, () => {
+        validationBubble = new ValidationBubble(window);
```

Comment:
We really only need one validation bubble. Even with multiple forms, we only need one because we're always showing the bubble on one field at a time. I don't really see reasons to have multiple instances here.